docs/streaming-programming-guide.md

```diff
@@ -1415,6 +1415,185 @@ Note that the connections in the pool should be lazily created on demand and tim
 
 ***
 
+## Accumulator and Broadcast
+
+Accumulator and Broadcast cannot be recovered from checkpoint in Spark Streaming. If you enable checkpointing and use Accumulator or Broadcast as well, you'll have to create lazily instantiated singleton instances for Accumulator and Broadcast so that they can be restarted on driver failures. This is shown in the following example.
```

Comment:
I believe its called "Broadcast variable"

http://spark.apache.org/docs/latest/programming-guide.html#broadcast-variables

Might be a good idea to put links to the Spark programming guide